core/src/main/scala/kafka/coordinator/group/GroupCoordinator.scala

```diff
@@ -693,7 +694,6 @@ class GroupCoordinator(val brokerId: Int,
   }
 
   private def onMemberFailure(group: GroupMetadata, member: MemberMetadata) {
```

Comment:
We do a little more than `removeMemberFromGroup` here since we trigger possible state transition as well after removing it. Will try to figure out a better name.